lib/ui/text/paragraph_impl_txt.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/text/paragraph_impl_txt.h"
+#include "flutter/common/threads.h"
+#include "flutter/lib/ui/text/paragraph.h"
+#include "flutter/lib/ui/text/paragraph_impl.h"
+#include "lib/ftl/tasks/task_runner.h"
+
+#include "lib/txt/src/paragraph_constraints.h"
+
+namespace blink {
+
+ParagraphImplTxt::ParagraphImplTxt(std::unique_ptr<txt::Paragraph>& paragraph) {
+  m_paragraph = std::move(paragraph);
+}
+
+ParagraphImplTxt::~ParagraphImplTxt() {}
+
+double ParagraphImplTxt::width() {
+  return m_width;
+}
+
+double ParagraphImplTxt::height() {  // TODO.
```

Comment:
TODO comments should be in the body of the function and should have your username in parathesis.  For example:

// TODO(abarth): Implement this function.